exporter/mezmoexporter/exporter.go

```diff
@@ -0,0 +1,174 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mezmoexporter
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"sync"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+type mezmoExporter struct {
+	config   *Config
+	settings component.TelemetrySettings
+	client   *http.Client
+	wg       sync.WaitGroup
+}
+
+type MezmoLogLine struct {
+	Timestamp int64             `json:"timestamp"`
+	Line      string            `json:"line"`
+	App       string            `json:"app"`
+	Level     string            `json:"level"`
+	Meta      map[string]string `json:"meta"`
+}
+
+type MezmoLogBody struct {
+	Lines []MezmoLogLine `json:"lines"`
+}
+
+func newLogsExporter(config *Config, settings component.TelemetrySettings) *mezmoExporter {
+	var e = &mezmoExporter{
+		config:   config,
+		settings: settings,
+	}
+	return e
+}
+
+func (m *mezmoExporter) pushLogData(ctx context.Context, ld plog.Logs) error {
+	m.wg.Add(1)
```

Comment:
I don't think this `wg` is neccessary.  stop() just needs to terminate background goroutines.

Reply:
Still needs to be resolved.